Makefile

```diff
@@ -3,39 +3,39 @@ SOURCES_LEGACY=snapcraft_legacy tests/legacy
 
 .PHONY: autoformat-black
 autoformat-black:
-	tox run -e format
+	tox run -e format-black
 
 .PHONY: freeze-requirements
 freeze-requirements:
 	tools/freeze-requirements.sh
 
 .PHONY: test-black
 test-black:
-	tox run -e black
+	tox run -e test-black
 
 .PHONY: test-codespell
 test-codespell:
-	tox run -e codespell
+	tox run -e test-codespell
 
 .PHONY: test-isort
 test-isort:
-	tox run -e isort
+	tox run -e test-isort
```

Comment:
shouldn't this call `tox run -e lint-isort`? (same for the other linters)

Reply:
Hey so it turns out `git push` is a useful command that I forgot to run :upside_down_face: